pkg/logger/config.go

```diff
@@ -0,0 +1,56 @@
+package logger
+
+import (
+	"os"
+
+	"github.com/pkg/errors"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+// Config helps to construct a customized zap logger.
+type Config struct {
+	OutputPaths   []string
+	Level         zap.AtomicLevel
+	EncoderConfig zapcore.EncoderConfig
+}
+
+// Build constructs a logger and returns a logger.
+func (cfg Config) Build() (*zap.Logger, error) {
+	sink, err := cfg.openSinks()
+	if err != nil {
+		return nil, err
+	}
+
+	if cfg.Level == (zap.AtomicLevel{}) {
+		return nil, errors.New("missing Level")
+	}
+
+	logger := zap.New(cfg.buildCore(sink), cfg.buildOptions()...)
+	return logger, nil
+}
+
+func (cfg Config) buildOptions() []zap.Option {
+	opts := []zap.Option{}
+	// Enable development mode
+	opts = append(opts, zap.Development())
```

Comment:
what does `zap.Development()` do and why are we setting it?

Reply:
Development "changes the behavior of DPanicLevel and takes stack-traces more liberally." according the [documentation](https://pkg.go.dev/go.uber.org/zap#Config). It was enabled originally, although I am unsure if we need this seeing that we were disabling stack-traces.

Reply:
Umm yeah I don't think we need it then

Reply:
Removed